### ISSUE TITLE
Can't create new style rule when none is present

### ISSUE THREAD

**Issue by jmacura** [hslayers/hslayers-ng#2880]
# Bug

**Describe the bug**
When a layer is added to the map and it has no styles, neither the default ones, it is impossible to create any style rules because of "appRef.styleObj is undefined" error.

**To Reproduce**
1. Go to 'Datasources'
2. Add layer 'nějaký políčko u rostěnic' from Layman@hub.lesprojekt as a WFS
3. Open 'styler' panel
4. Try to create new style rule
5. See error

**Expected behavior**
No errors and new style rule created

**Screenshots**
![image](https://user-images.githubusercontent.com/5598693/162838368-589aea27-232b-4066-8fd7-810b82e4a7c4.png)

**Additional context**
The actual business logic issue might be that no default style is assigned. Layman WFS is worth the investigation – why it did not assign any default styling.
Still, such an error should not prevent new rule creation.

**Comment by jmacura** [comment]
Weird. After I have filed the issue, I can't replicate that...

**Comment by DailisLangovskis** [comment]
@jmacura I suppose adding ->
```
if (!appRef.styleObject) {
        appRef.styleObject = {name: 'untitled style', rules: []};
      }
```
after here -> https://github.com/hslayers/hslayers-ng/blob/develop/projects/hslayers/src/components/styles/styler.service.ts#L319, should resolve it being undefined

**Comment by fzadrazil** [comment]
There was this error log in Layman. It is complaining about an invalid style
https://hub.lesprojekt.cz/flower/task/ec203332-68eb-4fc9-aaef-bc72b169bcb6
